packages/server/lib/cloud/api/cy-prompt/get_cy_prompt_bundle.ts

```diff
@@ -10,7 +10,7 @@ import { verifySignatureFromFile } from '../../encryption'
 const pkg = require('@packages/root')
 const _delay = linearDelay(500)
 
-export const getCyPromptBundle = async ({ cyPromptUrl, projectId, bundlePath }: { cyPromptUrl: string, projectId: string, bundlePath: string }) => {
+export const getCyPromptBundle = async ({ cyPromptUrl, projectId, bundlePath }: { cyPromptUrl: string, projectId?: string, bundlePath: string }) => {
```

Comment:
Why is `projectId` no longer required?

Reply:
We decided to always download the bundle. That way if they add a project id later, etc. we don't need to redownload. It just simplifies things.